tests/integration/targets/cloudfront_distribution/tasks/main.yml

```diff
@@ -410,6 +410,23 @@
       that:
         - update_origin_to_s3_with_origin_access_and_with_custom_origin_config.failed
 
+  - name: Update distribution to use specific access identity
+    cloudfront_distribution:
+      distribution_id: "{{ distribution_id }}"
+      origins:
+        - id: "{{ targetBucket }}"
```

Comment:
`targetBucket` is not defined anywhere in the test suite, which causes this task to fail.  I think you may have meant to use `resource_prefix` here?

Reply:
You are correct :+1:

Reply:
I have updated my PR to use `resource_prefix` instead